lymph/core/events.py

```
@@ -43,7 +43,8 @@ def serialize(self):
 
 
 class EventHandler(Component):
-    def __init__(self, interface, func, event_types, sequential=False, queue_name=None, active=True, once=False):
+    def __init__(self, interface, func, event_types, sequential=False,
+                 queue_name=None, active=True, once=False, broadcast=False):
         super(EventHandler, self).__init__()
```

Let's raise an exception if ``once and broadcast``. An assert would be good enough.